bigtable-dataflow-import/src/main/java/com/google/cloud/bigtable/sequencefile/testing/BigtableTableUtils.java

```diff
@@ -0,0 +1,93 @@
+package com.google.cloud.bigtable.sequencefile.testing;
+
+import com.google.cloud.bigtable.dataflow.CloudBigtableConfiguration;
+import com.google.cloud.bigtable.hbase1_0.BigtableConnection;
+import com.google.cloud.bigtable.sequencefile.HBaseImportOptions;
+import com.google.common.collect.Sets;
+
+import org.apache.hadoop.hbase.Cell;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Admin;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.ResultScanner;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.Table;
+
+import java.io.IOException;
+import java.util.Set;
+
+/**
+ * Helper for creating, deleting and reading a Cloud Bigtable during tests. Each instance manages
```

Comment:
google3 practice is to put testing helpers in java/. Dataflow team seems to stick to this too: e.g.  com.google.cloud.dataflow.sdk.testing.SourceTestUtils
Is the convention for Google on Github different?